Feat(web-react): Introduce spacing style props to all components #DS-475 (#1194)
lmc-eu/spirit-design-system: packages/web-react/README.md

do offer control over layout and other aspects. In addition, you can use Spirit defined design tokens to ensure your
application conforms to design requirements, and is adaptive across platform scales and color schemes.

### Style props

All Spirit components accept a set of props that can be used to control their outer spacing. The props are:

- `margin`
- `marginTop`
- `marginRight`
- `marginBottom`
- `marginLeft`
- `marginX`
- `marginY`

These props accept a spacing token (eg. `space-100`), `auto` or an object with breakpoint keys and spacing token

> values or `auto`. We use these props to set global CSS utility classes on the root element of the component.

Reviewer: > We use these props to set global CSS utility classes on the root element of the component.

Then we must make it clear in the package README that one must also import `utilities.css` in order to use the spacing props. I wouldn't be shy and introduce it with ⚠️.

https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-react#usage

Reviewer: The same goes for the Twig package.

Examples:

```jsx
<Alert marginBottom="space-100" />
<Button marginX={{ mobile: 'space-100', tablet: 'space-200' }} />
```

> If you need more control over the styling of a component, you can use [escape hatches](#escape-hatches).

Reviewer: Just slightly different, but it better says what we mean, I think:

```suggestion
If you need even more control over the component styling, use [escape hatches](#escape-hatches).
```
